src/components/Navbar/Navbar.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+
+import Navbar from './Navbar.component';
+
+describe('Navbar', () => {
+  test('should contain search input', () => {
+    render(<Navbar />);
+
+    const searchInput = screen.getByPlaceholderText(/Search/i);
```

Comment:
In general, as you will only use this line for the `expect`, you can remove this `const` and do it directly on the `expect`